src/doc/unstable-book/src/compiler-flags/tls-model.md

```diff
@@ -0,0 +1,24 @@
+# `tls_model`
+
+The tracking issue for this feature is: None.
+
+------------------------
+
+Option `-Z tls-model` controls [TLS model](https://www.akkadia.org/drepper/tls.pdf) used to
+generate code for accessing `#[thread_local]` `static` items.
```

Comment:
Technically speaking, the TLS model is supposed to be a per-variable attribute rather than a global one. However we don't currently have such an attribute, only a global setting which sets the default TLS model for all variables.